test/specs.js

```diff
@@ -174,4 +174,29 @@ describe('Custom handler function', function () {
     })
       .to.throw('[module-alias] Expecting custom handler function to return path.')
   })
+
+    it('should return a path matching the alias', function () {
+        var expected = path.join(__dirname, 'src/bar/baz');
+        moduleAlias.addAlias('@root', expected);
+
+        var rootPath = moduleAlias.getAliasPath('@root');
+        expect(rootPath)
+            .to.equal(expected);
```

Comment:
Move `.to.equal` to the same line as `expect`?

Reply:
Done